modules/utxo-bin/src/commands.ts

```diff
@@ -100,19 +142,32 @@ export const cmdParse = {
       argv.fetchSpends = true;
     }
 
-    const parsed = getParser(argv).parse(tx, {
+    const parsed = getTxParser(argv).parse(tx, {
       status: argv.fetchStatus ? await fetchTransactionStatus(txid, network) : undefined,
       prevOutputs: argv.fetchInputs ? await fetchPrevOutputs(tx) : undefined,
       prevOutputSpends: argv.fetchSpends ? await fetchPrevOutputSpends(tx) : undefined,
       outputSpends: argv.fetchSpends ? await fetchOutputSpends(tx) : undefined,
     });
-    switch (argv.format) {
-      case 'json':
-        console.log(JSON.stringify(parsed, null, 2));
-        break;
-      case 'tree':
-        console.log(formatTree(parsed, { hide: argv.all ? [] : undefined }));
-        break;
-    }
+
+    console.log(formatString(parsed, argv));
   },
-};
+} as const;
+
+export const cmdParseAddress = {
+  command: 'parseAddress [address]',
+  aliases: ['address'],
+  describe: 'parse address',
+  builder(b: yargs.Argv<unknown>): yargs.Argv<ArgsParseAddress> {
+    return b
+      .option('network', { alias: 'n', type: 'string' })
+      .option('format', { choices: ['tree', 'json'], default: 'tree' } as const)
+      .option('convert', { type: 'boolean', default: false })
```

Comment:
Did you consider adding aliases for these other options?

Reply:
you mean like single letter aliases? I'll add them when the long options get inconvenient for me